homeassistant/components/solaredge_local/sensor.py

```diff
@@ -164,10 +164,3 @@ def update(self):
         except (ConnectTimeout, HTTPError):
```

Comment:
Can we move the remaining attribute access above to a separate try... except block to make it clear what errors we are expecting in each case?

Reply:
Good one, changed it